README.md

```diff
@@ -144,6 +144,76 @@ Grype is a tool that can scan SBOMs for vulnerabilities and accepts them as inpu
 bomctl export urn:uuid:f360ad8b-dc41-4256-afed-337a04dff5db | grype
 ```
 
+### Deliver SBOMs to a remote OCI registry
+
+#### __Step 1: Understanding Push Flags__
+
+``` bash
+bomctl push --help
+```
+
+Similar to `export`, you have the option to specify the format and encoding of the pushed SBOM(s).
+Review flags for encoding and format [here](#step-1-understanding-export-flags)
+
+#### __Step 2: Push__
+
+Setup:
+
+Push the original SBOM to a "remote" oci registry. In this exercise, the env variable used `BOMCTL_PORT_URL` is already set in the environment, so the commands  
+can be copied/pasted as is. If you want a more interactive view, you can switch to the ports tab of the open terminal and hit the globe icon next to the  
+`Zot Registry` row. This will open the deployed Zot Registry we are using for our OCI registry endpoint. Based on [this](https://oras.land/docs/quickstart/) example from Oras.  
+
+__Note:__ Using this example registry is only necessary to make these exercises standalone, if you have a personal oci registry available, that can be used too.  
+        For authentication to those registries. a `.netrc` file needs to be created and a flag `--netrc` should be added to the push command.
```

Comment:
Should also specify that a `.netrc` file should have permissions of `rw-------` or `u=rw,go=` or `0600`

```suggestion
        For authentication to those registries, a `.netrc` file needs to be created and a flag `--netrc` should be added to the `push` command.
```